Repairs/app/src/main/res/layout/activity_dashboard.xml

```diff
@@ -64,4 +92,106 @@
         android:orientation="vertical"
         app:layout_constraintGuide_begin="1495dp" />
 
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/allJobs"
+        android:layout_width="806dp"
+        android:layout_height="250dp"
+        android:layout_marginTop="10dp"
+        android:layout_marginBottom="136dp"
+        android:background="@drawable/layout_bg_nopadding"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.482"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/notableJobs">
+
+        <View
+            android:id="@+id/divider4"
+            android:layout_width="804dp"
+            android:layout_height="1dp"
+            android:layout_marginTop="72dp"
+            android:background="?android:attr/listDivider"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/allJobs_Title"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="20dp"
+            android:text="All Jobs"
+            android:textSize="24sp"
+            app:layout_constraintBottom_toTopOf="@+id/divider4"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent" />
+
+        <View
+            android:id="@+id/divider5"
+            android:layout_width="1dp"
+            android:layout_height="150dp"
+            android:layout_marginStart="268dp"
+            android:layout_marginTop="84dp"
+            android:background="?android:attr/listDivider"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <View
+            android:id="@+id/divider6"
+            android:layout_width="1dp"
+            android:layout_height="150dp"
+            android:layout_marginTop="84dp"
+            android:layout_marginEnd="268dp"
+            android:background="?android:attr/listDivider"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/textView2"
```

Comment:
Give this a better id name, and change all of the rest of the ids that have generic names